src/ray/raylet/design_docs/task_states.rst

```diff
@@ -0,0 +1,36 @@
+A task can be in one of the following states:
```

Comment:
Let's move this to the top level directory under `design_docs/task_states.rst`.

Reply:
how about we put it into a subdirectory of ray/doc? Like ray/doc/design for example

Reply:
On the other hand, if we want them to be in the documentation page, then we need them to be under `doc/source/`, but that can be done later.